snuba/migrations/snuba_migrations/system/0001_migrations.py

```diff
@@ -37,11 +39,41 @@ def __backwards_local(self) -> Sequence[operations.Operation]:
             )
         ]
 
+    def __forwards_dist(self) -> Sequence[operations.Operation]:
+        return [
+            operations.CreateTable(
```

Comment:
Any reason not to update the `__forwards_local` method to use operations.CreateTable as well and to reuse the same table definition except for the engine ?
The generated code would not change and I think there is value in not duplicating the schema of the table unless there is a reason for having differences between local and distributed.

Reply:
Updated to create table